lib/primitive/lib/Cardano/Wallet/Primitive/Types/Tx/TxOut/Gen.hs

```diff
@@ -54,11 +54,16 @@ genTxOut :: Gen TxOut
 genTxOut = TxOut
     <$> genAddress
     <*> genTokenBundleSmallRange `suchThat` tokenBundleHasNonZeroCoin
+    <*> pure Nothing
```

Comment:
I think we should generate a wider range of values here than just `Nothing`.

Are there already generators for the `Script` type, and if so, can we import one of them and use it here?